lib/datastore/request.js

```diff
@@ -351,6 +368,10 @@ DatastoreRequest.prototype.insert = function(entities, callback) {
  * See below for examples of both approaches.
  *
  * @param {module:datastore/query} q - Query object.
+ * @param {object=} options - Optional configuration.
+ * @param {boolean} options.strongConsistency - Use strong read consistency. For
+ *     eventual consistency, use `false` for this value. If not specified,
+ *     default values are chosen by Datastore for the operation.
```

Comment:
It may be nice to add a comment here that you can't just set strongConsistency to strong to make an operation strongly consistent. You can only use this to make a strongly consistent operation eventual.